code/modules/surgery/organs/eyes.dm

```diff
@@ -183,17 +183,21 @@
 	name = "fung-eye"
 	desc = ""
 
-/obj/item/organ/eyes/night_vision/elf
-	name = "elf eyes"
+// Standardized darksight eyes
+
+/obj/item/organ/eyes/night_vision/full_darksight
+	name = "darksight eyes"
 	desc = ""
 	see_in_dark = 14
 
-/obj/item/organ/eyes/halfelf
-	name = "half-elf eyes"
+/obj/item/organ/eyes/half_darksight
+	name = "half-darksight eyes"
 	desc = ""
 	see_in_dark = 7
 	lighting_alpha = LIGHTING_PLANE_ALPHA_NV_TRAIT
```

Comment:
please change LIGHTING_PLANE_ALPHA_NV_TRAIT to LIGHTING_PLANE_ALPHA_MOSTLY_VISIBLE

View distance in darkness is already governed by see_in_dark, and the current lighting alpha does so little that it might as well not be there at all.